docs/LLS_Metallicities-Wotta/UVB_HM05.dat

```diff
@@ -0,0 +1,453 @@
+# The UVB spectrum (quasars+galaxies model) HM05
```

Comment:
I'm not sure what this file is for, but I feel it should
go in pyigm/data/euvb

Please make it so.